docs/cloud/pouta/persistent-volumes.md

```diff
@@ -22,6 +22,19 @@ is the size of the volume.
 
 ![Create persistent Volmume](../../img/create-volume-horizon.png)
 
+!!! Warning "Avoid non-ASCII character in name or description allowed"
+    This is a know bug in the volume system. Volume creation will fail
+    if its name or description contains any non-ASCII characters (e.g., ä, ö, å, é, à, ñ, [CJK characters](https://en.wikipedia.org/wiki/CJK_characters), ...):
+
+    ![Unable to create volume](../img/Unable-to-create-volume.png)
+
+    And the volume will be stuck in "Creating":
```

Comment:
The spaces in front of this sentence probably need to be removed, otherwise the sentence cannot be seen (see the preview)